src/main/java/com/eternalcode/parcellockers/gui/MainGUI.java

```diff
@@ -56,11 +61,11 @@ public void showMainGUI(Player player) {
         gui.addSlotAction(40, event -> event.getView().close());
         gui.addSlotAction(20, event -> {
             event.getView().close();
-            new ParcelListGUI(Bukkit.getServer(), this.miniMessage, this.config, this.repository).showParcelListGUI(player);
+            new ParcelListGUI(this.plugin, this.plugin.getServer(), this.miniMessage, this.config, this.parcelRepository, this.parcelLockerRepository).showParcelListGUI(player);
```

Comment:
co to za dziwna technika pobierania servera, wyrzuć podawanie pluginu oraz dodaj to jak człowiek jako pole Server i przekaż przez konstruktor.